tools/export/exporters.py

```diff
@@ -196,6 +197,7 @@ def scan_and_copy_resources(self, prj_paths, trg_path, relative=False):
             self.config_header = self.toolchain.MBED_CONFIG_FILE_NAME
             config.get_config_data_header(join(trg_path, self.config_header))
             self.config_macros = []
+            self.resources.inc_dirs.append(".")
```

Comment:
Actually, the trg_path is an absolute path to the .temp directory where the project files are generated. I did verify that the `Makefile` and `mbed_config.h` always live in the same directory.